src/FMBot.Bot/Services/IndexService.cs

```diff
@@ -158,7 +175,12 @@ public async Task<GuildUser> GetOrAddUserToGuild(Persistence.Domain.Models.Guild
                         UserName = discordGuildUser.Nickname ?? discordGuildUser.Username
                     };
 
-                    const string sql = "INSERT INTO guild_users (guild_id, user_id, user_name, bot) VALUES (@guildId, @userId, @userName, false) " +
+                    if (guild.WhoKnowsWhitelistRoleId.HasValue)
+                    {
+                        guildUserToAdd.WhoKnowsWhitelisted = discordGuildUser.RoleIds.Contains(guild.WhoKnowsWhitelistRoleId.Value);
+                    }
+
+                    const string sql = "INSERT INTO guild_users (guild_id, user_id, user_name, bot, who_knows_whitelisted) VALUES (@guildId, @userId, @userName, false, @whoKnowsWhitelisted) " +
```

Comment:
This has a hardcoded value for `bot` to false, but I think this method also gets used when servers add a bot to their server?

Reply:
This was the behavior pre-fork, and it seems like this method is only ever called in command handlers.

Reply:
![image](https://user-images.githubusercontent.com/20448879/123060172-7e485b80-d3d8-11eb-9181-bdf8c25ce145.png)